stdlib/3/inspect.pyi

```diff
@@ -132,17 +140,17 @@ class Parameter:
                  *,
                  default: Any = ...,
                  annotation: Any = ...) -> None: ...
-    empty = ...  # type: Any
-    name = ...  # type: str
-    default = ...  # type: Any
-    annotation = ...  # type: Any
-
-    kind = ...  # type: _ParameterKind
-    POSITIONAL_ONLY = ...  # type: _ParameterKind
-    POSITIONAL_OR_KEYWORD = ...  # type: _ParameterKind
-    VAR_POSITIONAL = ...  # type: _ParameterKind
-    KEYWORD_ONLY = ...  # type: _ParameterKind
-    VAR_KEYWORD = ...  # type: _ParameterKind
+    empty: Any
```

Comment:
These and the constants below (`POSITIONAL_ONLY` etc.) should have `= ...` to indicate that they're set on the class.

Reply:
Done (commit amended & force push)

Reply:
Actually, looks like `empty` still needs its `= ...`. The others are instance attributes and don't need it.

Reply:
Done (with force push :))

Reply:
`Signature.empty` is class attribute too